README.md

```diff
@@ -53,113 +56,72 @@ Flags:
+To view your default chain, run:
 
 ```
-default_chain: osmosis
- chains:
-   cosmoshub:
-     key: default
-     chain-id: cosmoshub-4
-     rpc-addr: https://cosmoshub-4.technofractal.com:443
-     grpc-addr: https://gprc.cosmoshub-4.technofractal.com:443
-     account-prefix: cosmos
-     keyring-backend: test
-     gas-adjustment: 1.2
-     gas-prices: 0.01uatom
-     key-directory: /Users/lenscrafters/.lens/keys
-     debug: false
-     timeout: 20s
-     output-format: json
-     sign-mode: direct
-   juno:
-     key: default
-     chain-id: juno-1
-     rpc-addr: https://rpc-juno.itastakers.com:443
-     grpc-addr: ""
-     account-prefix: juno
-     keyring-backend: test
-     gas-adjustment: 1.2
-     gas-prices: 0.01ujuno
-     key-directory: /Users/lenscrafters/.lens/keys
-     debug: false
-     timeout: 20s
-     output-format: json
-     sign-mode: direct
-   osmosis:
-     key: default
-     chain-id: osmosis-1
-     rpc-addr: https://osmosis-1.technofractal.com:443
-     grpc-addr: https://gprc.osmosis-1.technofractal.com:443
-     account-prefix: osmo
-     keyring-backend: test
-     gas-adjustment: 1.2
-     gas-prices: 0.01uosmo
-     key-directory: /Users/lenscrafters/.lens/keys
-     debug: false
-     timeout: 20s
-     output-format: json
-     sign-mode: direct
+ lens chains show-default
 ```
 
-When running a command, it will run the command for the defaulted chain. The defaulted chain can be found at the top of `~/.lens/config.toml` or by running `lens`. 
-
 To change your default, run: 
 
 ``` 
 lens chains set-default <chain_name>
 ```
 
-#### Keys
+### **Keys**
+Lens uses the keyring from the Cosmos-sdk. There is more information about it keyring [here](https://github.com/cosmos/cosmos-sdk/blob/master/crypto/keyring/doc.go). 
 
-Lens uses the keyring from the Cosmos-sdk. There is more information about it keyring [here](https://github.com/cosmos/cosmos-sdk/blob/master/crypto/keyring/doc.go). To add a key to lens run:
+To add a key to lens you have two options:
 
-``` 
-lens keys restore <key_name>
-```
+* `lens keys add` - This will add a key to you default chain and name it "default". You can optionally add a name as an argument. 
+* `lens keys restore <name>` - This will restore a key to your default chain. Replace '\<name\>' with a key name. This command will THEN ask for your mnemonic which is needed to restore and use it to boradcast transactions. 
+
+>❗️ NOTE: IF you name your key anything other than "default", you will need to manually change the `key:` value in your config to link key with chain.  
+>```bash
+>default_chain: cosmoshub
+>chains:
+>  cosmoshub:
+>    key: default #CHANGE THIS NAME
+>    chain-id: cosmoshub-4
+>    rpc-addr: https://cosmoshub-4.technofractal.com:443
+>    ...
+>```
+
+After adding or restoring a key, it should appear in your list by running: `lens keys list`, by default it will show the Cosmos Hub address. 
```

Comment:
```suggestion
After generating or restoring a key, it should appear in your list by running: `lens keys list`, by default it will show the Cosmos Hub address. 
```